web/modules/custom/dept_migrate/modules/dept_migrate_group_nodes/config/install/migrate_plus.migration.node_subtopic.yml

```diff
@@ -0,0 +1,89 @@
+id: node_subtopic
+label: 'Subtopic content type nodes'
+status: true
+dependencies: {  }
+migration_group: migrate_group_nodes
+migration_tags:
+  - dept_sites
+class: Drupal\migrate\Plugin\Migration
+field_plugin_method: alterFieldInstanceMigration
+source:
+  plugin: d7_node_uuid
+  node_type: subtopic
+  track_changes: true
+process:
+  langcode:
+    -
+      plugin: default_value
+      source: language
+      default_value: en
+  title: title
+  uid:
+    -
+      plugin: d7_user_lookup
+      source: node_uid
+  status: status
+  created: created
+  changed: changed
+  promote: promote
+  sticky: sticky
+  revision_uid:
+    - plugin: d7_user_lookup
+      source: revision_uid
+  revision_log: log
+  revision_timestamp: timestamp
+  body:
+    -
+      plugin: get
+      source: body
+    -
+      plugin: str_replace
+      regex: true
+      search: /(<p>[<br>|\s]*&nbsp;<\/p>)/im
+      replace: ''
+  body/0/format:
+    -
+      plugin: static_map
+      bypass: true
+      source: body/0/format
+      map:
+        filtered_html: basic_html
+        filtered_html_with_no_images: basic_html
+        filtered_html_with_templates: basic_html
+        filtered_html_with_tokens: basic_html
+        html_for_admins: full_html
+        paste_format: plain_text
+        plain_text: plain_text
+  field_parent_subtopic:
+    -
+      plugin: get
+      source: field_parent_subtopic
+  field_parent_topic:
+    -
+      plugin: get
+      source: field_parent_topic
+  field_summary:
+    -
+      plugin: get
+      source: field_summary
+  field_banner_image:
+    -
+      plugin: sub_process
+      source: field_banner_image
+      process:
+        target_id: fid
```

Comment:
This might need updating (see node_news migration) to pull the correct file id based on a lookup plugin. Otherwise it's not guaranteed to be a match as files will be added at both D7 and D9 sides during the progressive migration of site content.

Reply:
Correctly did this within topic and forgot about this field, nice catch.